### ISSUE TITLE
Request for Repository Access

### ISSUE THREAD

**Issue by Vedp9984** [virtual-labs/exp-pathloss-shadowing-iiith#6]
Hi,
 I am ved, and I am currently working on virtual labs. I need access to this repository to [reason for access, e.g., contribute to the project, review the code, etc.]. Could you please grant me the necessary permissions to all wireless communication experiments?

My GitHub is https://github.com/Vedp9984 .

Thank you!

Best regards,
ved prakash maurya(IIIT H)

**Comment by priya100raman** [comment]
Are you working for Prof Praful ? 
Please ask him to send a mail to priya@vlabs.ac.in and ravi.kiran@vlabs.ac.in with you in cc to confirm the same. 

Then follow the process listed in the Requesting Access to Existing Repositories for New Developers section

in the [Onboarding process document](https://vlead.vlabs.ac.in/development/#onboarding-process) to request access.

**Comment by Vedp9984** [comment]
yes mam I am working for Prof Praful.I will ask him to confirm the same

**Comment by Vedp9984** [comment]
@ravikiran2020 
Can you please provide me developer access to all the repos of this lab(wireless communication)?
Name : VED PRAKASH MAURYA
github handle: https://github.com/Vedp9984
Email:  ved.maurya@students.iiit.ac.in

**Comment by ravikiran2020** [comment]
@Vedp9984 To gain access to the lab repositories, please comment on the relevant lab repository creation issue in the Engineer Forum. You can find the specific issue here:
https://github.com/virtual-labs/engineers-forum/issues/918
